New prow docs after upgrade (#3674)
kyma-project/test-infra: docs/prow/prow-monitoring.md

Install the following tools:

- kubectl
- [jsonnet](https://jsonnet.org) and [jsonnet-bundler](https://github.com/jsonnet-bundler/jsonnet-bundler) installed

## Create dashboards configmaps

The steps assume you have completed the prerequisites.

1. Make sure your kubectl context is set to a correct cluster.

> 2. Go to the [prow/cluster/components/monitoring](../../prow/cluster/components/monitoring) directory and run command:

Reviewer: ```suggestion
2. Go to the [prow/cluster/components/monitoring](../../prow/cluster/components/monitoring) directory and run the command:
```

```shell
jb install
```
This will download all required dependencies for generating Grafana dashboards.

> 2. Once the command is done run the following commands:

Reviewer: ```suggestion
2. Once the command is done, run the following commands:
```

```shell
make generate_dashboards
make apply_configmaps
```
Those commands will generate json dashboards to the `dashboards_out` directory and apply them as configmaps to the kubernetes cluster.

> 3. Once everything is done run `make clean` to remove the generated json files.

Reviewer: ```suggestion
3. Once everything is done, run `make clean` to remove the generated json files.
```

## Apply monitoring deployments

> 1. If needed generate new Grafana password using a command `openssl rand -hex 12` and paste it as `password` value in the [`grafana_secret.yaml`](../../prow/cluster/components/monitoring/grafana_secret.yaml) file.

Reviewer: ```suggestion
1. If needed, generate a new Grafana password with command `openssl rand -hex 12` and paste it as `password` value in the [`grafana_secret.yaml`](../../prow/cluster/components/monitoring/grafana_secret.yaml) file.
```

> 2. Apply Prow namespace and grafana deployment YAMLs in the following order:

Reviewer: ```suggestion
2. Apply Prow namespace and Grafana deployment YAMLs in the following order:
```

```
prow_monitoring_namespace.yaml
grafana_rbac.yaml
grafana_configmaps.yaml
grafana_secret.yaml
grafana_deployment.yaml
monitoring_kyma-prow_managedcertificate.yaml
grafana_expose.yaml
```

> This will create grafana instance which will be exposed under https://monitoring.build.kyma-project.io.

Reviewer: ```suggestion
This will create a Grafana instance that will be exposed under https://monitoring.build.kyma-project.io.
```

3. Apply prometheus operator deployment YAMLs in the following order:
```
prometheus_operator_rbac.yaml
prometheus_operator_deployment.yaml
```

> 4. Apply Prow prometheus configuration YAMLs in the following order:

Reviewer: ```suggestion
4. Apply the Prow Prometheus configuration YAMLs in the following order:
```
